### PR TITLE
Added vue-animejs plugin and a component to test it

### PR DESCRIPTION
**Pull request by DHvyas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ pranjul139
 ```
 npm install
 ```
+```
+npm install --save vue-animejs
```
Comment (stealthanthrax) [inline README.md:14]: This is not required, it is automatically added in the package.json as a dependency

File: src/components/static/Animation1.vue [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/animejs/2.2.0/anime.min.js"></script>
+<script src="./script.js"></script>
+<script>
+export default {
+  components: {},
+  mounted: function() {
+    this.$anime({
+      targets: "#lineDrawing .lines path",
+      strokeDashoffset: [anime.setDashoffset, 0],
```
Comment (stealthanthrax) [inline src/components/static/Animation1.vue:48]: replace this with [this.$anime.setDashoffset, 0] , else the code wouldn't compile.

File: src/components/static/Animation1.vue [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        </g>
+      </svg>
+    </div>
+  </div>
+</template>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/animejs/2.2.0/anime.min.js"></script>
+<script src="./script.js"></script>
+<script>
+export default {
```
Comment (stealthanthrax) [inline src/components/static/Animation1.vue:43]: give a name while exporting the component, 
```
export default {
 name: 'Animation1'
...
```

**Comment by DHvyas** [comment]
> @DHvyas , resolve the inline comments.
> 
> 
> 
> Also, you have not imported your animation in any of the `views` file and hence not rendering on screen.
> 
> 
> 
> To do the same , use `import Animation1 from "@/components/static/Animation1";`

I had done so initially but the component was not rendering as desired (without animation), hence I didn’t do so.

**Comment by stealthanthrax** [comment]
It was working fine for me. In which view where were you importing the component?
> I had done so initially but the component was not rendering as desired (without animation), hence I didn’t do so.

**Comment by stealthanthrax** [comment]
@DHvyas , the animation is working fine for me.

**Comment by stealthanthrax** [comment]
What is the expected behaviour?

**Comment by DHvyas** [comment]
> @DHvyas , the animation is working fine for me.

Yeah I think it was the ~~~this.$anime~~ in the export function why it didn’t render properly for me. Fixed now

**Comment by stealthanthrax** [comment]
Alright then. Finalise your PR and we'll merge it then. 🖖

**Comment by stealthanthrax** [comment]
@DHvyas , is this the final version of this PR?

**Comment by DHvyas** [comment]
> @DHvyas , is this the final version of this PR?

Yeah I will commit other components as another PR

**Comment by stealthanthrax** [comment]
Alright then.

**Pull request merged**